dvc/command/version.py

```diff
@@ -1,30 +1,111 @@
 from __future__ import unicode_literals
 
+import os
+import psutil
 import platform
 import argparse
 import logging
+import uuid
 
+from dvc.repo import Repo
 from dvc.command.base import CmdBaseNoRepo, append_doc_link
 from dvc.version import __version__
+from dvc.exceptions import DvcException, NotDvcRepoError
+from dvc.system import System
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdVersion(CmdBaseNoRepo):
     def run(self):
-        info = (
-            "DVC version: {dvc_version}\n"
-            "Python version: {python_version}\n"
-            "Platform: {platform_type}"
-        ).format(
-            dvc_version=__version__,
-            python_version=platform.python_version(),
-            platform_type=platform.platform(),
-        )
+        dvc_version = __version__
+        python_version = platform.python_version()
+        platform_type = platform.platform()
+        root_directory = os.getcwd()
+
+        try:
+            root_directory = Repo.find_root()
+
+            info = (
+                "DVC version: {dvc_version}\n"
+                "Python version: {python_version}\n"
+                "Platform: {platform_type}\n"
+                "Filesystem Type: {filesystem_type}\n"
+                "Cache: {cache}"
+            ).format(
+                dvc_version=dvc_version,
+                python_version=python_version,
+                platform_type=platform_type,
+                filesystem_type=self.get_fs_type(
+                    os.path.abspath(root_directory)
+                ),
+                cache=self.get_linktype_support_info(),
+            )
+
+        except NotDvcRepoError:
+            info = (
```

Comment:
No need to copypaste this information again. You could simply append link info if it is available.